src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--start-day--sun calendar--month-length--28">
```

Comment:
You should use here `<ul>` tag with `<li>` to use more semantic HTML 

```suggestion
      <ul class="calendar calendar--start-day--sun calendar--month-length--28">
        <li class="calendar__day"></li>
        <li class="calendar__day"></li>
        <li class="calendar__day"></li>
        <li class="calendar__day"></li>
        <li class="calendar__day"></li>
        <li class="calendar__day"></li>
```